azurelinuxagent/common/utils/archive.py

```diff
@@ -38,12 +38,18 @@
 
 MAX_ARCHIVED_STATES = 50
 
+# These files we'll move - deleting them in the old location
 CACHE_PATTERNS = [
     re.compile("^(.*)\.(\d+)\.(agentsManifest)$", re.IGNORECASE),
     re.compile("^(.*)\.(\d+)\.(manifest\.xml)$", re.IGNORECASE),
     re.compile("^(.*)\.(\d+)\.(xml)$", re.IGNORECASE)
 ]
 
+# These files we'll copy only and leave them in the original location
```

Comment:
If you call this function everytime the FastTrackExtensionConfig is modified then you can move these files from COPY to MOVE too